### DISCUSSION TITLE
Hot stepper motor

### DISCUSSION THREAD

**Discussion by RUtarah** [Jyers/Marlin#214] [Q&A] [answered]
Hi, due to hotter stepper motor I try to change current from 800 for XYZ to 580 and Extruder from 800 to 650. but it seem nothing happen if i adjust that currents value. This value I refers from my old Ender 3s (mboard skr e3 dip 1.1) and implement it to mboard 4.2.2 E3V2. My old Ender 3s much cooler than E3V2. Can we adjust that current for E3V2 or it already fixed?

**Comment by Schnabulation** [comment, marked as answer]
The stepper drivers on the Ender 3 V2 board are in standalone mode - you can not adjust the vref currents in the firmware. You have to do it manually directly on the board.

Also: The Ender 3 V2 has different stepper drivers than the Ender 3 - it needs different vref values.

**Reply by RUtarah** [reply]
thanks sir! finally i got the answer. And is it difficult to do it? I am still new in this 3D field

**Reply by Schnabulation** [reply]
I was pretty much a noob in electronics before owning the printer and I was able to do it myself. :-)

You need a multimeter though: https://www.youtube.com/watch?v=hC4J29rxPlQ
Also, very important: This is a video for the Ender 3 - the vref values on this video are WRONG! The correct ones are these: https://marlin.crc.id.au/faq/Ender%203%20V2/#vref

**Reply by RUtarah** [reply]
Nice! I also like to try and learn new things. Oh that value like my Ender 3s. I will do it now! XO. Thanks sir!

**Reply by RUtarah** [reply]
Finally my stepper motor more cold now. I adjust all vref with 0.96 and all working good no skipping step. Thanks mate!

**Reply by Barackazarc** [reply]
Um may I ask here? You set this vref for every stepper? I am asking that because im struggling with the same issue like for a month now.
Thank you for your reply
